### PR TITLE
[BE] enable UFMT for `torch/masked/`

### PR DESCRIPTION
**Pull request by XuehaiPan**
Part of #123062 

- #123062

### CODE REVIEW

File: torch/masked/_ops.py [hunk 1]
```diff
@@ -1,15 +1,12 @@
-
 import warnings
-
-# A workaround to support both TorchScript and MyPy:
 from typing import Any, List, Optional, Tuple, TYPE_CHECKING, Union
 
 import torch
-from torch import Tensor
-from torch.masked import as_masked_tensor, is_masked_tensor, MaskedTensor
```
Comment (XuehaiPan) [inline torch/masked/_ops.py:9]: Cyclic imports are caused by this line. Changed them to where they are defined.

```diff
- from torch.masked import as_masked_tensor, is_masked_tensor, MaskedTensor
+ from torch.masked.maskedtensor.core import is_masked_tensor, MaskedTensor
+ from torch.masked.maskedtensor.creation import as_masked_tensor
```

**Review by cpuhrsch — APPROVED** [review]
